src/raster/r.hand/testsuite/test_r_hand.py

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/env python3
+
+############################################################################
+#
+# MODULE:       test_r_hand.py
+# AUTHOR:       Corey T. White, OpenPlains Inc.
+# PURPOSE:      Performs Height Above Nearest Drainage (HAND) analysis.
+# COPYRIGHT:    (C) 2025 OpenPlains Inc.
+#               This program is free software under the GNU General
+#               Public License (>=v2). Read the file COPYING that
+#               comes with GRASS for details.
+#
+#############################################################################
+
+
+import grass.script as gs
+from grass.gunittest.case import TestCase
+from grass.gunittest.gmodules import SimpleModule
+from grass.gunittest.main import test
+
+
+class TestHand(TestCase):
+    elevation = "elev_lid792_1m"
```

Comment:
I suggest to use "elevation" in tests, my goal is to limit the layers to basic dataset when possible.

Reply:
I switch to `elev_lid792_1m` because the tests ran faster, but I can change the region to reduce the spatial extent and set a higher threshold for r.watershed.